recipes/civetweb/all/conanfile.py

```diff
@@ -82,20 +82,32 @@ def package(self):
                 os.remove(os.path.join(bin_folder, bin_file))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.components["_civetweb"].names["cmake_find_package"] = "civetweb"
+        self.cpp_info.components["_civetweb"].names["cmake_find_package_multi"] = "civetweb"
+        self.cpp_info.components["_civetweb"].names["pkg_config"] = "civetweb"
+        self.cpp_info.components["_civetweb"].libs = ["civetweb"]
         if self.settings.os == "Linux":
-            self.cpp_info.system_libs.extend(["dl", "rt", "pthread"])
-            if self.options.with_cxx:
-                self.cpp_info.system_libs.append("m")
+            self.cpp_info.components["_civetweb"].system_libs.extend(["dl", "rt", "pthread"])
         elif self.settings.os == "Macos":
-            self.cpp_info.frameworks.append("Cocoa")
-            self.cpp_info.sharedlinkflags = self.cpp_info.exelinkflags
-            self.cpp_info.defines.append("USE_COCOA")
+            self.cpp_info.components["_civetweb"].frameworks.append("Cocoa")
+            self.cpp_info.components["_civetweb"].sharedlinkflags = self.cpp_info.exelinkflags
```

Comment:
This line seems a little odd =? why is it required?

Reply:
seems to be a leftover from the previous approach to handle Apple Frameworks (via `-framework` link flags). shouldn't be necessary here
```suggestion
```

Reply:
I don't know, it was in original file here: https://github.com/civetweb/civetweb/blob/master/conanfile.py

I checked on my laptop (macos) - looks like it works and the self.cpp_info.exelinkflags list is empty anyway.

Removed it